packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -1,17 +1,17 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
```

Comment:
FYI, you'll need to fix these lints before you submit

Reply:
Oh wow I must have accidentally added these via VSCode

Reply:
These are still here FYI